content/en/news/2016-04-10-results-of-the-2nd-gd.md

```diff
@@ -0,0 +1,200 @@
+donduct the Second General Discsussion ([original schedule][1]). In
+this article, I summarise what was discussed, what was decided, and
+what not was decided.
+
+First, here’s the (anchored) link to the full [chatlogs][2], so that
+you can read what exactly happened if you want.
+
+Voting Rules Changes
+--------------------
+
+The team voted on the following change proposal:
+
+<!-- Note to translators: Please do not translate the proposal
+below. It is a verbatim copy of the proposal sent to the ML. -->
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+                                 Art. 1
+
+§2(2) is changed to “The vote requires that two thirds of the team
+members voting are in favour of adding the new person to the team.”.
+
+§3(1)(Nr.3)(2nd sentence) is changed to “This requires a two thirds
+majority of those voting, and the candidate for removal is not allowed
+to participate in the vote.”
+
+In §20 the words “of the team members” are replaced with “of all
+existing team members”.
+
+                                 Art. 2
+
+A new §5a is introduced after §5 with the following text:
+
+§5a IRC Voting Announcement
+
+(1) For votings held in IRC — as an exception to § 5(2)(Nr.3) — it is
+sufficient to announce all available voting options in IRC prior to
+taking the votes.
+
+(2) If at least three team members are available in IRC and are not just
+idling, it is possible to waive the two-week preparation interval and
+vote immediately. The waiver requires all non-idling team members in the
+chatroom to agree on it.
+
+(3) Any team member can challenge a voting made under the conditions
+of (2) above until two weeks after the declaration of results have
+passed by approaching the project lead accordingly. If the voting is
+challenged, it is treated the same way as if it failed (§12).
+
+                                 Art. 3
+
+These changes enter into force immediately after the Declaration of
+Results of the vote for accepting them.
+
+Any votes running at the moment these changes enter into force are
+transferred to the changed version of the rules and do not remain under
+the old version.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Of 8 team members allowed to vote, 7 participated and voted in favour
+of it. Thus, the required 2/3 majority was reached and the changes are
+in force now.
+
+The newly compiled voting rules document is available [online in the
+documents reopsitory now][3].
+
+Team Additions
+--------------
+
+Under the new rules, the team then continued with voting on adding
+kirbyfan and skarfester to the team. [Both][4] [votes][5]
+succeeded. As of now, the two just need to accept their membership on
+GitHub.
+
+Bugtracker Software
+-------------------
+
+After [a fairly long discussion][7], all team members decided in consent to
+chose [Redmine][6] as our future bugtracking system as it has a lot of
+nice and useful features (like especially dependencies between tasks
+as metadata with useful summaries in parent tickets) and it’s written
```

Comment:
This sentence is kind of long... What if you end it after the closing paren, then start a new one beginning with **In addition**?